src/styles/blocks/header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+.header__list-item {
+  position: relative;
+}
+
+.header__list-item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.header__list-item-link {
+  display: flex;
+  height: 60px;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  align-items: center;
+  color: $card__text-color;
+}
+
+.is-active {
+  color: $nav__list-item-link-color;
+}
+
+.header__logo {
+  display: flex;
+  align-items: center;
+}
+
+.header__list-item-link:hover {
+  color: $nav__list-item-link-color;
+}
+
+.header__list-item-link.is-active::after {
```

Comment:
Don't increase specificity if no needed and use nesting with `is-active` class styles